specs/Concepts.md

```diff
@@ -31,12 +31,16 @@
 
 <ul>
 
-**`1.b.1.` Test Forme** — an "imposed" sheet containing one or more test elements.
```

Comment:
Can we do what ISO does here?

Look up "Test Forme" [here](https://www.sovsib.ru/color/iso12647_en.pdf)

/cc @FranzSigg

Reply:
Yes we can.